### ISSUE TITLE
Adding new package to backend 'externals'

### ISSUE THREAD

**Issue by josh--newman** [fredguile/react-redux-es2015-starter-kit#4]
First of all, thanks for sharing this starter kit. It's almost exactly what I've been looking for, especially the backend webpack config.

My question is, how do I add a new 'external' package to be bundled up with the webpack.backend.js config? Is it as simple as adding it as a dependency in `package.json`? I've done that and it seems it's not getting packaged.

I'm quite new to webpack so please forgive any obvious things I'm missing here.

Thanks!

**Comment by fredguile** [comment]
Thanks! I've put some effort into making this work...

So yes, simply add new npms using "npm install", they will be available from your backend code (as they are in node_modules).

The only problem I had to address was to avoid that Webpack "swallows" them into the built backend code. That's unnecessary to transform anything that lives into node_modules, so here's the way I excludes node_modules to be processed by Webpack:

``` javascript
const externals =
  chain(fs.readdirSync(path.resolve(__dirname, '../node_modules')))
    .filter(mod => mod !== '.bin')
    .reduce(externalsReducer, manualExternals)
    .value();
```

Even with this hack, I still got issues when importing modules that has a "/" in my code:

``` javascript
import fileContent from 'shared-store/file'; // Webpack will swallow that module into the backend build
```

The way I found to prevent this is to explicitly declare this "nested" dependency in webpack.backend.js here:

``` javascript
// references that we manually put in the externals
const manualExternals = reduce([
  'shared-store/file',
  'gofer/hub',
  'react-dom/server',
], externalsReducer, {});
```

I agree that's a pain to think about this. Fortunately not all imports use slashes.

Feel free to suggest/PR me a better solution! :)
